release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+==== Insights Runtime Extractor (Technology Preview)
+
+In {product-title} 4.18, the Insights Operator introduces a new workload data collection feature, _Insights Runtime Extractor_ (`InsightsRuntimeExtractor`), to help Red{nbsp}Hat better understand the workload of your containers.
+Available as a Technology Preview, the `InsightsRuntimeExtractor` feature gathers and sends runtime workload data to Red{nbsp}Hat. 
+Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.
+For more information about how to enable the feature, see _Enabling features using feature gates_.
```

Comment:
In addition to the suggestions above, consider breaking this up and adding a link to "Enabling features using feature gates":

```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers.

Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. 
Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.

For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

Reply:
Done.  
Good idea. Note that, previous peer review suggestions requested that my xrefs were removed from release notes and added to the Additional Information section of the assembly instead.